src/_guides/language/language-tour.md

```diff
@@ -1129,7 +1129,7 @@ is sometimes referred to as _arrow_ syntax.
 
 A function can have two types of parameters: _required_ and _optional_.
 The required parameters are listed first, followed by any optional parameters.
-Optional parameters can be _named_ or _positional_.
+Optional parameters can be either _named_ or _positional_.
```

Comment:
This is basically all wrong for Null Safety. We can have required named parameters placed after optional named parameters.

How about:

> A function can have any number of _required positional_ parameters. 
> These may be followed either by _optional positional_ parameters, 
> or by _named_ parameters. 
> A named parameter can be either optional or `required`.

If this is describing pre-Null Safety behavior, then I'd say:

> A function can have any number of _required_ parameters. 
> These may be followed either by _optional positional_ parameters, 
> or by _optional named_ parameters.